source/jormungandr/jormungandr/interfaces/v1/__init__.py

```diff
@@ -49,8 +51,17 @@ def add_common_status(response, instance):
         response['status']['realtime_proxies'].append(realtime_proxy.status())
 
     response['status']['street_networks'] = []
-    for sn in instance.get_all_street_networks():
-        response['status']['street_networks'].append(sn.status())
+    street_networks = instance.get_all_street_networks()
```

Comment:
The output of a method should always be the same type, whatever the configuration is